packages/dcos-integration-test/extra/conftest.py

```diff
@@ -21,6 +21,31 @@
 LOGIN_PW = os.getenv('DCOS_LOGIN_PW')
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers', 'order(n): mark test to run in Nth order.'
+        'Negative numbers represent position from the end.'
+        'EG: 1, 2, 3, ... unordered ... -3, -2, -1')
+
+
+def pytest_collection_modifyitems(session, config, items):
+    """Reorders test using order mark
+    """
+    first = []
+    unordered = []
+    last = []
+    for item in items:
+        if not hasattr(item.obj, 'order'):
```

Comment:
> if the [1, 2, 3, 0, -1, -2, -3] is wanted
What I want is [1,2,3,-3,-2,-1] so that order(1) clearly stays the first item and order(-1) clearly stays the last item. If we go order [3,2,1,0,-1,-2,-3] and then some other test must come second my test_cluster_is_up is test must be changed to be order(2) and the new test order(1) (need to edit the entire stack of orders to guarantee timing) or I would call my new test order(0), and then we are locked into using negatives thereafter and not to mention its a little awkward. Furthermore, my order(-1) test will always be at least tied for the end. with [3,2,1,0,-1,-2,-3], I need to set something like order(-sys.maxint - 1)